src/Symfony/Component/Notifier/Bridge/Telegram/Tests/TelegramTransportTest.php

```diff
@@ -330,4 +330,106 @@ public function testSendWithMarkdownShouldEscapeSpecialCharacters()
 
         $transport->send(new ChatMessage('I contain special characters _ * [ ] ( ) ~ ` > # + - = | { } . ! to send.'));
     }
+
+    public function testSendPhotoWithOptions()
+    {
+        $response = $this->createMock(ResponseInterface::class);
+        $response->expects($this->exactly(2))
+            ->method('getStatusCode')
+            ->willReturn(200);
+
+        $content = <<<JSON
+            {
+                "ok": true,
+                "result": {
+                    "message_id": 1,
+                    "from": {
+                        "id": 12345678,
+                        "is_bot": true,
+                        "first_name": "YourBot",
+                        "username": "YourBot"
+                    },
+                    "chat": {
+                        "id": 1234567890,
+                        "first_name": "John",
+                        "last_name": "Doe",
+                        "username": "JohnDoe",
+                        "type": "private"
+                    },
+                    "date": 1459958199,
+                    "photo": [
```

Comment:
file_id is always the same?

do we really need so many files for the test case or is one enough?

Reply:
Yes, i think telegram generate smaller version of the image passed.

All file_id are the same for each version of the same file.

I wrote the test with the same response i've got when i tested, but i can keep only one item